### ISSUE TITLE
Email Error

### ISSUE THREAD

**Issue by pblair12** [pblair12/mosaic-groups#27]
Some emails are not being sent.  I checked the logs and I am seeing this

Check the logs here: https://papertrailapp.com/systems/mosaicgroups/events?q=Error+sending+email

```
Dec 20 12:05:35 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) f193sm11223383qhe.6 - gsmtp] 
Dec 20 12:05:36 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) a66sm2159224qkb.20 - gsmtp] 
Dec 20 12:05:37 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) a66sm2159277qkb.20 - gsmtp] 
Dec 20 12:05:39 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) d64sm11219794qgd.48 - gsmtp] 
Dec 20 12:05:41 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) x72sm11361583qkx.29 - gsmtp] 
Dec 20 12:05:41 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) v59sm11263097qge.38 - gsmtp] 
Dec 20 12:05:43 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) e134sm11237868qhc.49 - gsmtp] 
Dec 20 12:05:43 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) g90sm10535656qgg.45 - gsmtp] 
Dec 20 12:05:43 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) l124sm11214469qhc.24 - gsmtp] 
Dec 20 12:05:44 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) b6sm9426065qkh.12 - gsmtp] 
Dec 20 12:05:49 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) s131sm11227153qhs.11 - gsmtp] 
Dec 20 12:05:49 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) u202sm11356233qka.10 - gsmtp] 
Dec 20 12:05:50 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) f126sm11357592qkb.43 - gsmtp] 
Dec 20 12:05:51 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) d6sm11360710qkb.13 - gsmtp] 
Dec 20 12:05:53 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) t90sm4082180qkt.8 - gsmtp] 
Dec 20 12:05:54 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) p101sm6847557qkp.36 - gsmtp] 
Dec 20 12:06:02 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) m46sm11275233qgd.47 - gsmtp] 
Dec 20 12:06:02 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) w32sm11285041qgw.6 - gsmtp] 
Dec 20 12:06:02 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) 6sm4947911qhg.17 - gsmtp] 
Dec 20 12:06:02 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) c3sm11264515qge.12 - gsmtp] 
Dec 20 12:06:05 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) d188sm8452806qkb.9 - gsmtp] 
Dec 20 12:06:05 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) j67sm3421212qgj.35 - gsmtp] 
Dec 20 12:06:11 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) v129sm11240992qhb.0 - gsmtp] 
Dec 20 12:06:11 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) f83sm5618601qkb.25 - gsmtp] 
Dec 20 12:06:17 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) y200sm11337787qka.48 - gsmtp] 
Dec 20 12:06:17 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) z8sm11281308qgz.7 - gsmtp] 
Dec 20 12:06:23 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) f66sm11381003qkb.5 - gsmtp] 
Dec 20 12:06:23 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) e184sm11354840qkb.40 - gsmtp] 
Dec 20 12:06:25 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) t47sm11267157qgt.28 - gsmtp] 
Dec 20 12:06:26 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) z51sm11250355qgd.36 - gsmtp] 
Dec 20 12:06:49 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) q12sm11382819qki.15 - gsmtp] 
Dec 20 12:06:49 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) d64sm11221378qgd.48 - gsmtp] 
Dec 20 12:07:02 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) g10sm11359531qkb.26 - gsmtp] 
Dec 20 12:07:06 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) z21sm9033873qge.18 - gsmtp] 
Dec 20 12:07:07 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) p33sm11258500qge.14 - gsmtp] 
Dec 20 12:07:07 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) x72sm11363546qkx.29 - gsmtp] 
Dec 20 12:07:07 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) b80sm11358938qkb.17 - gsmtp] 
Dec 20 12:07:07 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) f91sm11256252qgf.32 - gsmtp] 
Dec 20 12:07:12 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) u135sm11368421qka.22 - gsmtp] 
Dec 20 12:07:12 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) d62sm11235154qga.41 - gsmtp] 
Dec 20 12:07:13 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) p33sm11258632qge.14 - gsmtp] 
Dec 20 12:07:13 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Try again later, closing connection. (MAIL) b6sm9428073qkh.12 - gsmtp] 
Dec 20 16:06:09 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Connection expired, try reconnecting. q18sm11443929qkq.47 - gsmtp] 
Dec 20 16:06:09 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Connection expired, try reconnecting. 6sm5135481qhg.17 - gsmtp] 
Dec 20 16:06:13 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Connection expired, try reconnecting. g130sm11402005qkb.28 - gsmtp] 
Dec 20 16:11:16 mosaicgroups app/web.1:  Error sending email { [SenderError: Mail from command failed - 421 4.7.0 Connection expired, try reconnecting. l124sm11383326qhc.24 - gsmtp] 
```

**Comment by pblair12** [comment]
@jeffklassen says:

> google's smtp server has a limit per emails it can send per hour/day
It's reasonably low... something like a few hundred/day
for my higher rate/email apps, i usually use my domain name provider as the SMTP server
usually with a domain name you get a few email accounts
and i use one of those
yeah and given the rate its sending these Audit messages....

**Comment by pblair12** [comment]
Closed it by mistake ;)

**Comment by pblair12** [comment]
Fixed with #38
